dozer-ingestion/src/connectors/postgres/schema/helper.rs

```diff
@@ -319,12 +319,17 @@ impl SchemaHelper {
         let is_column_used_in_index: bool = row.get(3);
         let replication_type_int: i8 = row.get(5);
         let type_oid: u32 = row.get(6);
-        let typ = Type::from_oid(type_oid);
 
-        let typ = typ.map_or_else(
-            || Err(InvalidColumnType(column_name.clone())),
-            postgres_type_to_dozer_type,
-        )?;
+        // TODO: workaround - in case of custom enum
+        let typ = if type_oid == 28862 {
```

Comment:
I think the same workaround is needed in `xlog_mapper` otherwise streaming won't work

Reply:
gotcha

Reply:
```rust
            // TODO: workaround - in case of custom enum
            let type_oid = column.type_id() as u32;
            let typ = if type_oid == 28862 {
                Type::VARCHAR
            } else {
                Type::from_oid(type_oid).ok_or_else(|| {
                    PostgresSchemaError::InvalidColumnType(column_name.to_string())
                })?
            };

            columns.push(TableColumn {
                name: column_name.to_string(),
                flags: column.flags(),
                r#type: typ,
                column_index,
            })
```
this would be the corresponding changes in the `xlog_mapper`, let me know if this looks good

Reply:
Write test and find out

Reply:
Just confirming before we move on to the testing

Reply:
Looks good to me, but I'm not sure without testing